qcodes/tests/test_snapshot.py

```diff
@@ -0,0 +1,39 @@
+"""
+Test module for snapshots of instruments and parameters
+"""
+
+from qcodes.tests.instrument_mocks import SnapShotTestInstrument
+
+
+def test_snapshot_skip_params_update(request):
+    """
+    Test that params_to_skip_update works as expected, in particular that only
+    the parameters given by that variable are skipped
```

Comment:
i find it a bit confusing that `params_to_skip_update` argument is not explicitly used in this test. i expected to see many `inst.snapshot(...)` with different combinations of `update` and `params_to_skip_update` values. can the test be improved so that the tested cases are more explicit and not hidden inside `SnapShotTestInstrument`?